### DISCUSSION TITLE
[Question] Questions about Delta Join supported by Fluss

### DISCUSSION THREAD

**Discussion by Au-Miner** [apache/fluss#82] [General]
### Search before asking

- [X] I searched in the [issues](https://github.com/alibaba/fluss/issues) and found nothing similar.

### Fluss version

master

### Minimal reproduce step

DeltaJoin feature positioning

### What doesn't meet your expectations?

I did not find any implementation of DeltaJoin in flux. Where is it located

### Anything else?

none

### Are you willing to submit a PR?

- [X] I'm willing to submit a PR!

**Comment by polyzos** [comment]
We should move this as a question to discussions
https://github.com/alibaba/fluss/discussions

@Au-Miner Delta Join is not implemented yet, but it's on the [roadmap](https://alibaba.github.io/fluss-docs/roadmap/#flink-integration)
